### PR TITLE
Replace fact categories with tags

### LINKED ISSUE #80: Switch from categories to tags
We want to switch from each fact having a single category to having an array of tags. This can be displayed on the large fact card with a set of small chips at the bottom or top.

### PR DESCRIPTION
**Pull request by UtkarshKr007**
We have been sorting facts into categories like 'habitat' and 'behavior', but not all facts can be easily categorized into a single criteria. A category like 'habitat' can contain facts about both 'plants' and 'homes' associated with pollinators which is why we are switching to tags. Tags will allow a fact to be associated with multiple criteria while being general enough, but not as encompassing as categories.

### CODE REVIEW

**Comment by UtkarshKr007** [comment]
Worked with @NicolasRobertson on the previous commit. Included the co-author in the commit, but he is not being credited for some reason.

File: src/components/large-fact-card.js [hunk 1]
```diff
@@ -80,20 +93,37 @@ export default function LargeFactCard({factTitle, factImage, factSource, factHTM
...
-              <IconButton aria-label="fact info" onClick={handleDialogClickOpen}>
-                <InfoOutlinedIcon />
-              </IconButton>
-            </Grid>
+
+          <Grid item>
+            <IconButton aria-label="fact info" onClick={handleDialogClickOpen}>
+              <InfoOutlinedIcon />
+            </IconButton>
+          </Grid>
```
Comment (floogulinc) [inline src/components/large-fact-card.js]: This should all be in a single `<Grid item>` or we get this:

image

**Review by floogulinc — CHANGES_REQUESTED** [review]
I have made some tweaks that are too much to do in suggestions here so I made another branch and a PR to this branch: https://github.com/mn-pollinators/pollinator-facts/pull/111

File: src/components/tag.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import { makeStyles } from '@material-ui/core/styles';
+import Chip from '@material-ui/core/Chip';
+import { GatsbyLink } from "gatsby-theme-material-ui"
+
+const tagStyles = makeStyles({
+  tag: {
+    backgroundColor: '#D3D3D3',
```
Comment (floogulinc) [inline src/components/tag.js]: I don't think we need to set this slightly darker grey. The default is fine.

**Comment by floogulinc** [comment]
Example from today's discussion:
```ts
allPosts.filter(post => searchTags.every(st => post.tags.includes(st)))
```

File: src/components/large-fact-card.js [hunk 3]
```diff
@@ -28,12 +28,18 @@ const factCardStyles = makeStyles({
   },
   body: {
    "& p": {
-    margin: 0
-   }
+    margin: 0,
+   },
+  },
+  inlineTags: {
+    marginBottom: '8px',
```
Comment (floogulinc) [inline src/components/large-fact-card.js]:
```suggestion
    marginBottom: '12px',
```

**Review by floogulinc — APPROVED** [review]
I think we might want to tweak what tags some of the facts have (eg should flowers get both `habitat` and `plant` (or maybe `flower`)) but other than that it looks good.

**Pull request merged**